src/main/java/games/strategy/util/TimeManager.java

```diff
@@ -1,20 +1,63 @@
 package games.strategy.util;
 
-import java.text.DateFormat;
-import java.text.SimpleDateFormat;
-import java.util.Date;
-import java.util.SimpleTimeZone;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.FormatStyle;
 
 public class TimeManager {
   /**
-   * Replacement for Date.toGMTString();
+   * Replacement for {@code Date.toGMTString();}.
    *
-   * @param date The Date being returned as String
+   * @param instant The {@link Instant} being returned as String
    * @return formatted GMT Date String
    */
-  public static String getGMTString(final Date date) {
-    final DateFormat dateFormat = new SimpleDateFormat("dd MMM yyyy HH:mm:ss z");
-    dateFormat.setTimeZone(new SimpleTimeZone(0, "GMT"));
-    return dateFormat.format(date);
+  public static String getGMTString(final Instant instant) {
+    return DateTimeFormatter.ofPattern("d MMM yyyy HH:mm:ss 'GMT'").withZone(ZoneOffset.UTC).format(instant);
```

Comment:
How important is it to preserve the output here?  I see that you've changed from `dd` (leading zero for single-digit day-in-month) to `d` (no leading zero for single-digit day-in-month)?  Is that going to screw anything up?  I had assumed you hard-coded the `GMT` for this very reason (presumably the first iteration of the new code that used `z` was producing `UTC`, `Z`, or something else other than `GMT`).

If it's only used for display purposes, then I guess there's no issue.

Currently only `HeadlessGameServer` calls this method to produce a status message.  I'm assuming there's no code out there that attempts to parse the status message, right? :smile: